Fastlane/testing_lanes.rb

```diff
@@ -58,7 +58,7 @@
       suppress_xcode_output: false,
       buildlog_path: ENV['BITRISE_DEPLOY_DIR'], # By configuring `BITRISE_DEPLOY_DIR` we make sure our build log is deployed and available in Bitrise.
       prelaunch_simulator: true,
-      xcargs: "-clonedSourcePackagesDirPath #{source_packages_dir} -parallel-testing-enabled NO -retry-tests-on-failure -test-iterations 3",
+      xcargs: "-clonedSourcePackagesDirPath #{source_packages_dir} -parallel-testing-enabled NO -retry-tests-on-failure -test-iterations 3 -test-timeouts-enabled -maximum-test-execution-time-allowance 60",
```

Comment:
Maybe it is also worth it for us to start investing in making the test ready for parallel testing.

Reply:
That's a great suggestion, I've added that to our backlog: https://wetransfer.atlassian.net/browse/TMOB-2117